### PR TITLE
 Add workaround to https://bugzilla.redhat.com/show_bug.cgi?id=1776594 

### PR DESCRIPTION
**Pull request by anpingli**

### CODE REVIEW

File: features/logging/collector.feature [hunk 1]
```diff
@@ -9,6 +9,17 @@ Feature: collector related tests
...
     Given evaluation of `cluster_logging('instance').fluentd_ready_pods.map(&:ip)` is stored in the :collector_pod_ips clipboard
+    #A workaround to https://bugzilla.redhat.com/show_bug.cgi?id=1776594
+    And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .operations.* |
+      | op           | DELETE                                                                                                                                                                                                                     |
+    Then the step should succeed
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | project.* |
+      | op           | DELETE                                                                                                                                                                                                                     |
```
Comment (QiaolingTang) [inline features/logging/collector.feature:20]: missing `|` in the end.

File: features/logging/collector.feature [hunk 2]
```diff
@@ -9,6 +9,17 @@ Feature: collector related tests
   Scenario: All nodes logs are sent to Elasticsearch
     Given the master version == "4.1"
     Given evaluation of `cluster_logging('instance').fluentd_ready_pods.map(&:ip)` is stored in the :collector_pod_ips clipboard
+    #A workaround to https://bugzilla.redhat.com/show_bug.cgi?id=1776594
+    And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .operations.* |
+      | op           | DELETE                                                                                                                                                                                                                     |
```
Comment (QiaolingTang) [inline features/logging/collector.feature:16]: missing | in the end.

File: features/logging/collector.feature [hunk 3]
```diff
@@ -42,6 +53,17 @@ Feature: collector related tests
...
     #And evaluation of `cluster_logging('instance').fluentd_ready_pods.map(&:node_ip)` is stored in the :node_ips clipboard
+    #A workaround to https://bugzilla.redhat.com/show_bug.cgi?id=1776594
+    And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .operations.* |
+      | op           | DELETE                                                                                                                                                                                                                     |
+    Then the step should succeed
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .project.* |
+      | op           | DELETE                                                                                                                                                                                                                     |
```
Comment (pruan-rht) [inline features/logging/collector.feature:64]: @anpingli the ending `|` all seems to have extra spaces.  Please correct.

File: features/logging/collector.feature [hunk 4]
```diff
@@ -42,6 +53,17 @@ Feature: collector related tests
       | component=<%= cb.collection_type %> |
     And evaluation of `@pods.map {|n| n.node_ip}.uniq` is stored in the :node_ips clipboard
     #And evaluation of `cluster_logging('instance').fluentd_ready_pods.map(&:node_ip)` is stored in the :node_ips clipboard
+    #A workaround to https://bugzilla.redhat.com/show_bug.cgi?id=1776594
+    And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .operations.* |
+      | op           | DELETE                                                                                                                                                                                                                     |
```
Comment (QiaolingTang) [inline features/logging/collector.feature:60]: missing | in the end.

File: features/logging/collector.feature (hunk 3 above)
Comment (QiaolingTang) [inline features/logging/collector.feature:64]: missing | in the end.

File: features/logging/collector.feature (hunk 2 above)
Comment (anpingli) [reply features/logging/collector.feature:16]: Stranger. There is '|' is in the end.   try to remove the spaces between DELETE and |.

File: features/logging/collector.feature [hunk 5]
```diff
@@ -42,12 +54,25 @@ Feature: collector related tests
...
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .project.* |
+      | op           | DELETE     |
+    Then the step should succeed
+    #Workaround end
     And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
     Given I get the ".operations" logging index information from a pod with labels "es-node-master=true"
     And the expression should be true> cb.index_data['docs.count'] > "0"
-    And I perform the HTTP request on the ES pod with labels "es-node-master=true":
+    Given I wait up to 300 seconds for the steps to pass
```
Comment (QiaolingTang) [inline features/logging/collector.feature:71]: @anpingli an example of step `I wait up to xxx seconds for the steps to pass:`  https://github.com/openshift/verification-tests/blob/master/features/logging/cluster_logging_operator.feature#L29-L36

File: features/logging/collector.feature [hunk 6]
```diff
@@ -9,12 +9,24 @@ Feature: collector related tests
...
+    Then the step should succeed
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | project.* |
+      | op           | DELETE    |
+    Then the step should succeed
+    #Workaround end
     And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
     Given I get the ".operations" logging index information from a pod with labels "es-node-master=true"
     And the expression should be true> cb.index_data['docs.count'] > "0"
+    Given I wait up to 300 seconds for the steps to pass
```
Comment (anpingli) [inline features/logging/collector.feature:26]: @QiaolingTang   Test Pass. https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/81744/.   A further question: how to quota the following steps in  one block as we have use the quota Mark Already.

Comment (QiaolingTang) [reply features/logging/collector.feature:26]: @anpingli I checked the logs, looks like the job didn't run with these changes, I'm not sure what happened here. 

> how to embrace the following steps into one block as we have use the quota Mark Already.
Not sure, let me try.

Comment (QiaolingTang) [reply features/logging/collector.feature:26]: @anpingli maybe we need to consider to update these steps: https://github.com/openshift/verification-tests/pull/899/files#diff-b2d95cc514cea4d6b51728cfc37cfbdcR77-R80

File: features/logging/collector.feature [hunk 7]
```diff
@@ -42,25 +53,37 @@ Feature: collector related tests
...
     #And evaluation of `cluster_logging('instance').fluentd_ready_pods.map(&:node_ip)` is stored in the :node_ips clipboard
+    #A workaround to https://bugzilla.redhat.com/show_bug.cgi?id=1776594
     And I wait for the ".operations" index to appear in the ES pod with labels "es-node-master=true"
-    Given I get the ".operations" logging index information from a pod with labels "es-node-master=true"
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .operations.* |
+      | op           | DELETE        |
+    Then the step should succeed
+    Given I perform the HTTP request on the ES pod with labels "es-node-master=true":
+      | relative_url | .project.* |
```
Comment (QiaolingTang) [inline features/logging/collector.feature:63]: `project.*`

**Pull request merged**